airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/write/StandardDestinationTasks.kt

```diff
@@ -0,0 +1,107 @@
+package io.airbyte.cdk.write
+
+/**
+ * These tasks together are sufficient to drive the StandardDestination lifecycle.
+ */
+
+fun taskList(d: StandardDestination) = TaskList(listOf(
```

Comment:
This might be a good use case for
`fun StandardDestination.taskList() = TaskList ...`

Can't remember what this pattern is called. In any case it makes for more readable code (`d` becomes `this`, with all that that implies) without forcing you to define a method on the object.

Reply:
this is just there so I can show what the approach would look like. This is experimental code I pushed over the weekend. I was about to post in #proj-new-java-cdk about the change in status.

Reply:
Oh I just mentioned this in case you weren't familiar with this kotlin idiom, that's all. I didn't mean to be prescriptive here.